src/nvim/ex_cmds2.c

```diff
@@ -744,40 +744,40 @@ void dbg_breakpoint(char_u *name, linenr_T lnum)
 
 static char_u   *profile_fname = NULL;
 
-/*
- * ":profile cmd args"
- */
+/// ":profile cmd args"
+///
+/// @param eap Pointer to the command arguments structure.
```

Comment:
I think just the `:profile cmd args` is necessary here. Anyone wanting information on how this should work can run `:h :profile`, and the contents of `eap` is required reading for all `ex_cmds[2].c` functions.